### ISSUE TITLE
"Black produced different code on the second pass of the formatter": unwrapping multiple parentheses

### ISSUE THREAD

**Issue by dizzy57** [psf/black#836]
Operating system: MacOS Mojave 10.14.4 (18E226)
Python version: Python 3.7.3
*Black* version: 19.5b0 (c2c2f7203869)
Does also happen on master: yes

Minified reproducer:
```python
return (("xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"))
```

Black diff:
```diff
--- source
+++ first pass
@@ -1,2 +1,4 @@
-return (("xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"))
+return (
+    "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+)

--- first pass
+++ second pass
@@ -1,4 +1,2 @@
-return (
-    "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
-)
+return "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
```

**Comment by Jma353** [comment]
I believe I have a fix for this.  Will put a PR up by tomorrow likely to address this.

**Comment by Jma353** [comment]
Proposed fix in #961
